crates/editor/src/editor_tests.rs

```diff
@@ -5236,11 +5236,27 @@ async fn test_select_all_matches(cx: &mut gpui::TestAppContext) {
     init_test(cx, |_| {});
 
     let mut cx = EditorTestContext::new(cx).await;
+
+    // Test carrot-only selections
```

Comment:
Should this be "caret"? Or does "carrot" have a meaning here that I'm not aware of? 😄 

```suggestion
    // Test caret-only selections
```

Reply:
🥕 .... 💀